lnrpc/invoicesrpc/addinvoice.go

```diff
@@ -180,6 +184,33 @@ func AddInvoice(ctx context.Context, cfg *AddInvoiceConfig,
 		)
 	}
 
+	// Check if any open channel has enough inbound bandwidth to receive an
+	// invoice of the specified amount
+	if invoice.CheckInboundBandwidth {
+		inboundBandwidth := false
+
+		openChannels, err := cfg.ChanDB.FetchAllOpenChannels()
```

Comment:
we might also consider only checking active channels, but I think this is okay (less aggressive).

Reply:
I did not think about this but it's an easy change. As this check is optional I guess we can be strict here and make it require an active channel.

Reply:
just added this on a fixup commit